bin/docker-entrypoint.sh

```diff
@@ -7,16 +7,14 @@ then
   EDGE_PORT=4566
 fi
 
-# FIXME: remove with 2.0
-# the Dockerfile creates .pro-version file for the pro image. When trying to activate pro features with any other
-# version, an error is printed.
-if [[ $LOCALSTACK_API_KEY ]] && [[ ! -f /usr/lib/localstack/.pro-version ]]; then
+# the Dockerfile creates .pro-version file for the pro image and .bigdata-pro-version for the bigdata image.
+# When trying to activate pro features with any other version, a warning is printed.
+if [[ $LOCALSTACK_API_KEY ]] && ! compgen -G /usr/lib/localstack/.*pro-version >/dev/null; then
```

Comment:
FYI: I just double-checked that the compgen approach with the glob works just fine. 👍🏽